controllers/openstackserver_controller_test.go

```diff
@@ -85,19 +84,6 @@ var defaultPortsStatus = []infrav1.PortStatus{
 	},
 }
 
-var createDefaultPort = func(r *recorders) {
```

Comment:
This means that we are now not testing any case where a port is created. All test cases find already existing ports. We need to cover also the case when ports are created.